postcss.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  plugins: [require('autoprefixer'), require('postcss-gap-properties')],
```

Comment:
Unfortunately it doesn't seem to be working. I'll send a screenshot of how it looks like on Safari 13 on Discord. It might also need webpack configuration which might be not easily available as we're using a ready-made build configuration by using Vite.